### ISSUE TITLE
non-canonical predicates

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#398]
For this query:
```
{ "message": {"query_graph": {
  "edges": {
    "e0": {
      "object": "n1",
      "predicates": [
        "biolink:gene_associated_with_condition",
        "biolink:condition_associated_with_gene",
        "biolink:genetic_association"
      ],
      "subject": "n0"
    }
  },
  "nodes": {
    "n0": {
      "categories": [
        "biolink:Gene"
      ],
      "ids": [
        "NCBIGene:727897"
      ]
    },
    "n1": {
      "categories": [
        "biolink:Disease"
      ]
    }
  }
}}}
```
There are edges in the knowledge_graph that have the non-canonical "biolink:condition_associated_with_gene" predicate.

**Comment by maximusunc** [comment]
Should be fixed once #400 is deployed

**Comment by maximusunc** [comment]
This has been deployed to dev. Please re-open if you run into this issue again.
